Trimble.Modus.Components/Controls/Toast/TMToastContents.xaml.cs

```diff
@@ -46,46 +46,46 @@ private void SetTheme(String toastTheme)
-                TextColor = ResourcesDictionary.ColorsDictionary(ColorsConstant.ToastTextBlue);
+                ToastBackground = (Color)BaseComponent.colorsDictionary()[toastTheme];
+                LeftIconSource = ImageSource.FromFile(ImageConstants.BlueInfoIcon);
+                TextColor = (Color)BaseComponent.colorsDictionary()["ToastTextBlue"];
                 break;
 
             case ToastTheme.Secondary:
-                ToastBackground = ResourcesDictionary.ColorsDictionary(ColorsConstant.TertiaryButton);
-                LeftIconSource = ImageSource.FromResource("Trimble.Modus.Components.Images.secondary.png");
-                TextColor = ResourcesDictionary.ColorsDictionary(ColorsConstant.Black);
+                ToastBackground = (Color)BaseComponent.colorsDictionary()[toastTheme];
+                LeftIconSource = ImageSource.FromFile(ImageConstants.SolidHelpIcon);
+                TextColor = (Color)BaseComponent.colorsDictionary()["Black"];
                 break;
 
             case ToastTheme.Danger:
-                ToastBackground = ResourcesDictionary.ColorsDictionary(ColorsConstant.DangerToastColor);
-                LeftIconSource = ImageSource.FromResource("Trimble.Modus.Components.Images.danger.png");
-                TextColor = ResourcesDictionary.ColorsDictionary(ColorsConstant.Black);
+                ToastBackground = (Color)BaseComponent.colorsDictionary()[toastTheme];
+                LeftIconSource = ImageSource.FromFile(ImageConstants.ToastDangerIcon);
+                TextColor = (Color)BaseComponent.colorsDictionary()["Black"];
                 break;
 
             case ToastTheme.Warning:
-                ToastBackground = ResourcesDictionary.ColorsDictionary(ColorsConstant.WarningToastColor);
-                LeftIconSource = ImageSource.FromResource("Trimble.Modus.Components.Images.warning.png");
-                TextColor = ResourcesDictionary.ColorsDictionary(ColorsConstant.Black);
+                ToastBackground = (Color)BaseComponent.colorsDictionary()[toastTheme];
+                LeftIconSource = ImageSource.FromFile(ImageConstants.WarningIcon);
+                TextColor = (Color)BaseComponent.colorsDictionary()["Black"];
                 break;
 
             case ToastTheme.Success:
-                ToastBackground = ResourcesDictionary.ColorsDictionary(ColorsConstant.SuccessToastColor);
-                LeftIconSource = ImageSource.FromResource("Trimble.Modus.Components.Images.input_valid_icon.png");
-                TextColor = ResourcesDictionary.ColorsDictionary(ColorsConstant.Black);
+                ToastBackground = (Color)BaseComponent.colorsDictionary()[toastTheme];
+                LeftIconSource = ImageSource.FromFile(ImageConstants.ValidIcon);
+                TextColor = (Color)BaseComponent.colorsDictionary()["Black"];
                 break;
 
             case ToastTheme.Default:

```

Comment:
We can remove this line since default method will be hit

Reply:
Done